cloud/pkg/cloudhub/servers/udsserver/uds.go

```diff
@@ -62,6 +63,16 @@ func (us *UnixDomainSocket) StartServer() error {
 		}
 	}
 
+	dir, err := filepath.Abs(filepath.Dir(addr))
+	if err != nil {
+		klog.Errorf("invalid dir of addr found: %v", err)
+	}
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
```

Comment:
ref: https://github.com/kubeedge/kubeedge/blob/6df871802fd63f2a0f0ce4b3ae8e2c015206237d/pkg/util/validation/check.go#L6

Reply:
Does `FileIsExist` fix this issue `failed to listen addr: listen unix //var/lib/kubeedge/kubeedge.sock: bind: no such file or directory`? @kadisi

Reply:
no, I suggest using a FileIsExist() method to determine whether  file or path is exists.

Reply:
If use `kubeedge/pkg/util/validation/check`, should move the dir creation there as well.
Minor issue, either looks fine to me.